cmd/server/assets/admin/users/show.html

```diff
@@ -0,0 +1,69 @@
+{{define "admin/users/show"}}
+{{$user := .user}}
+{{$stats := .stats}}
+
+<!doctype html>
+<html lang="en">
+<head>
+  {{template "head" .}}
+</head>
+
+<body id="users-show" class="tab-content">
+  {{template "admin/navbar" .}}
+
+  <main role="main" class="container">
+    {{template "flash" .}}
+
+    <h1>{{$user.Name}}</h1>
+    <p>
+      Here is information about the user.
```

Comment:
Maybe leave this paragraph out?

Reply:
I think having an explanatory subtitle is the pattern for all of the other pages. This one is pretty lame though - wish we had more to say.
I'm going to leave it for consistency